training/benchmarks/llava1.5_13b/deepspeed/configA800.py

```diff
@@ -0,0 +1,20 @@
+# Common arguments
+theoryflops = 312000000000000.0
+model_name_or_path = "/root/llava1.5_pretrain/data/LLaVA-Pretrain/checkpoints/vicuna-13b-v1.5"
+vision_tower = "/root/llava1.5_pretrain/data/LLaVA-Pretrain/openai/clip-vit-large-patch14-336"
```

Comment:
all path should be relative path to "data_dir". user can only set PERF_PATH and DATA_DIR writable, while other directories remain read-only